### ISSUE TITLE
Pull Request Comments not sent in webhook event

### ISSUE THREAD

**Issue by ICoppola-Copado** [go-gitea/gitea#15624]
- Gitea version (or commit ref):
- Git version:
- Operating system:
  
  
  
- Database (use `[x]`):
  - [x ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [X ] Yes : https://try.gitea.io/Icoppola/Test2/pulls/2/files
  [Webhook Response](https://webhook.site/#!/50eadfdf-838f-4f6c-a82d-8c69c07410c5/5709a97c-dc3e-492a-a8d0-c0ac37d4d521/1)
  - [ ] No
- Log gist:

## Description

When commenting inside of the modified files of a pull request, the event is not adding the message information in review > content

Steps to reproduce: 

1. Create a new repository
2. Add user two as collaborator
3. Setup a webhook Custom Events > "Pull request comment" event enabled 
4. Create a new file README.md in "master"
5. Create a new branch "feature1" out of master
6. Modify the file README.md in the "feature1" branch
7. Create a new pull request (1)
8. Login with user two
9. Go pull request (1) >> Files Changed >> Click in a line (+) >> Add a comment with this content "This is my comment content"
10. Check the webhook event response, will look like the screenshot below
...

## Screenshots

![image](https://user-images.githubusercontent.com/42381627/116081794-fa554a00-a69a-11eb-9a20-ec3e4bb8224f.png)

**Comment by KN4CK3R** [comment]
Which webhook type did you use?

**Comment by IC-Demo** [comment]
Hi, I used Gitea. 

These are the settings: 
[Update Webhook - Test2 - Gitea_ Git with a cup of tea.pdf](https://github.com/go-gitea/gitea/files/6384684/Update.Webhook.-.Test2.-.Gitea_.Git.with.a.cup.of.tea.pdf)

**Comment by KN4CK3R** [comment]
`content` gets filled the review comment and not the inline comment:
![grafik](https://user-images.githubusercontent.com/1666336/116266763-12e96100-a77c-11eb-8a74-0a9c7a9119c9.png)

But the inline comment is missing completly. If you create multiple inline "Start review" comments they are missing too. So I think the empty `content` is by design but the missing comment sections are a bug.

**Comment by IC-Demo** [comment]
That's true, thanks for the help regarding the `content` attribute. That's been clarified.

So the bug is that the inline comments are not sent in the PR event.
